great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py

```diff
@@ -131,16 +131,78 @@ class ExpectColumnQuantileValuesToBeBetween(ColumnExpectation):
+            "config_version": 1.0,
+            "variables": {
+                "quantiles": [
+                    0.25,
+                    0.5,
+                    0.75,
+                ],
+                "allow_relative_error": "linear",
+                "num_bootstrap_samples": 9999,
+                "false_positive_rate": 0.05,
+                "truncate_values": {
+                    "lower_bound": None,
+                    "upper_bound": None,
+                },
+                "round_decimals": 1,
+            },
+            "rules": {
+                "column_quantiles_rule": {
+                    "domain_builder": {
+                        "class_name": "ColumnDomainBuilder",
+                        "module_name": "great_expectations.rule_based_profiler.domain_builder",
+                    },
+                    "parameter_builders": [
+                        {
+                            "name": "quantile_value_ranges",
+                            "class_name": "NumericMetricRangeMultiBatchParameterBuilder",
+                            "module_name": "great_expectations.rule_based_profiler.parameter_builder",
+                            "metric_name": "column.quantile_values",
+                            "metric_domain_kwargs": "$domain.domain_kwargs",
+                            "metric_value_kwargs": {
+                                "quantiles": "$variables.quantiles",
+                                "allow_relative_error": "$variables.allow_relative_error",
+                            },
+                            "num_bootstrap_samples": "$variables.num_bootstrap_samples",
+                            "false_positive_rate": "$variables.false_positive_rate",
+                            "round_decimals": "$variables.round_decimals",
+                        }
+                    ],
+                    "expectation_configuration_builders": [
+                        {
+                            "expectation_type": "expect_column_quantile_values_to_be_between",
+                            "class_name": "DefaultExpectationConfigurationBuilder",
+                            "module_name": "great_expectations.rule_based_profiler.expectation_configuration_builder",
+                            "column": "$domain.domain_kwargs.column",
+                            "quantile_ranges": {
+                                "quantiles": "$variables.quantiles",
+                                "value_ranges": "$parameter.quantile_value_ranges.value.value_range",
+                            },
+                            "allow_relative_error": "$variables.allow_relative_error",
+                            "meta": {
+                                "profiler_details": "$parameter.quantile_value_ranges.details"
+                            },
+                        }
+                    ],
+                }
+            },
+        },
```

Comment:
Any reason this is a dict not a "ProfilerConfiguration"?

Reply:
@jcampbell In an earlier conversation with @anthonyburdi we agreed that it would be better to defer the instantiation until needed (because the `auto` flag can be `False`, meaning that the user does not want the automatic expectation kwargs estimation feature to be turned on).  Either will work -- the code would have to convert at some point in order to reconcile custom and default configurations.

Reply:
While I think we should not instantiate a `Profiler` object here we should instantiate a `ProfilerConfig` object instead of a dictionary to take advantage of the validation, documentation and guiding parameters of the ProfilerConfig class. Alex and I discussed and this will be changed before merge.

Reply:
@jcampbell This got done in a better way to use typed configuration object (hat tip to @donaldheppner and @anthonyburdi ) as well as per your suggestion to define the default profiler above.  Thanks.